pallets/runtime/tests/src/compliance_manager_test.rs

```diff
@@ -1417,7 +1243,7 @@ fn can_verify_restriction_with_primary_issuance_agent_we() {
     // Add compliance requirement that requires sender to be primary issuance agent (dynamic)
     // and receiver to be a specific random_guy_id.
     assert_ok!(ComplianceManager::add_compliance_requirement(
-        owner_origin,
+        owner.origin(),
         ticker,
         vec![Condition {
```

Comment:
Could you closure-ize the common parts here (take `TargetIdentity` as a parameter)?